lib/Steps/Integrations/ReactNative.ts

```diff
@@ -507,7 +511,7 @@ export SENTRY_PROPERTIES=sentry.properties
           // remove sentry properties export
           .replace(/^export SENTRY_PROPERTIES=sentry.properties\r?\n/m, '')
           .replace(
-            /^\/bin\/sh ..\/node_modules\/@sentry\/react-native\/scripts\/collect-modules.sh\r?\n/m,
+            /^\/bin\/sh -c.*?..\/node_modules\/@sentry\/react-native\/scripts\/collect-modules.sh"?\r?\n/m,
```

Comment:
I've missed this at first, but also the `-c` should go, so we match the prev version of the command.
```suggestion
            /^\/bin\/sh .*?..\/node_modules\/@sentry\/react-native\/scripts\/collect-modules.sh"?\r?\n/m,
```